imageio/imageio-tiff/src/main/java/com/twelvemonkeys/imageio/plugins/tiff/TIFFImageMetadata.java

```diff
@@ -1024,6 +1024,34 @@ else if ("HorizontalPixelSize".equals(nodeName)) {
             else if ("VerticalPixelSize".equals(nodeName)) {
                 yRes = Float.parseFloat(getAttribute(child, "value"));
             }
+            else if ("ImageOrientation".equals(nodeName)) {
+                switch (getAttribute(child, "value").toLowerCase()) {
```

Comment:
Can you also make this `null`-safe, in the odd case of malformed metatada, just in case?